Source/gravity/Gravity.cpp

```diff
@@ -574,27 +574,12 @@ Gravity::gravity_sync (int crse_level, int fine_level, const Vector<MultiFab*>&
       if ( gravity::direct_sum_bcs )
           fill_direct_sum_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
       else {
-          if (gravity::lnum >= 0) {
-              fill_multipole_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
-          } else {
-              int fill_interior = 0;
-              make_radial_phi(crse_level,*rhs[0],*delta_phi[crse_level],fill_interior);
-          }
-      }
-#elif (BL_SPACEDIM == 2)
-      if (gravity::lnum >= 0) {
           fill_multipole_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
-      } else {
-          int fill_interior = 0;
-          make_radial_phi(crse_level,*rhs[0],*delta_phi[crse_level],fill_interior);
       }
+#elif (BL_SPACEDIM == 2)
+      fill_multipole_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
 #else
-      if (gravity::lnum >= 0) {
-          fill_multipole_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
-      } else {
-          int fill_interior = 0;
-          make_radial_phi(crse_level,*rhs[0],*delta_phi[crse_level],fill_interior);
-      }
+      fill_multipole_BCs(crse_level,fine_level,amrex::GetVecOfPtrs(rhs),*delta_phi[crse_level]);
```

Comment:
what's the difference in this if?

Reply:
Nothing. It was already there, I just didn't remove it. I was going to clean that up later -- my next plan is to make the direct sum BCs work in <3D, and then we don't need any of this logic.